GitUI/UserControls/RevisionGrid.cs

```diff
@@ -323,7 +324,7 @@ public void SetInitialRevision(GitRevision initialSelectedRevision)
             _initialSelectedRevision = initialSelectedRevision != null ? initialSelectedRevision.Guid : null;
         }
 
-        private bool _isRefreshingRevisions = false;
```

Comment:
We risk descending into off topic here debating whether one should familiarise oneself with the language/tools/convensions before using them. Or discussing values of devtools like re#, stylecop, fxcop etc.
I'll roll this back